log/consul.go

```diff
@@ -0,0 +1,106 @@
+package log
+
+import (
+	"io"
+	"log"
+	"os"
+
+	hclog "github.com/hashicorp/go-hclog"
+)
+
+var consulLog = &consul{}
+
+// Consul return a consul compatible logger.
+func Consul() hclog.Logger {
```

Comment:
I think it would make more sense to create a different `logger` package.
It is a bit confusing when using `log.Consul()` to get a logger, while `logger.Consul()` it is more intuitive.

Reply:
I started actually doing that but, in the end, I realized that we do not need a separate package which will be exported by default. We need an adapter that in invisible to the outside world.
Having it in a different package created also a cyclic dependency as be wanted to set it up whenever the harvester logger is set up.
This way everything is under the covers and nothing is publicly visible.

Reply:
I get it. How about rename `Consul()` to `ConsulLogger()` then ?